example/foo.go

```diff
@@ -31,6 +31,7 @@ type FooResponse struct {
 	Instruction   Instruction            `json:"instruction"`
 	BsonPtr       *BsonID                `json:"bsonPtr,omitempty" example:"blah blah blah"`
 	RandomBool    bool                   `json:"randomBool,omitempty" example:"true"`
+	MyEnum        string                 `json:"myEnum" goasEnum:"value1,value2"`
```

Comment:
I think the convention is that the tag is the name of the package, so we should maybe do something like

```
goas:"enum=value1,value2"
```

Or we can hijack [go-playground/validator's oneof tag](https://pkg.go.dev/github.com/go-playground/validator/v10#hdr-One_Of):

```
validator:"oneof=value1 value2"
```

Reply:
@XieX I think we would then need to change how it parses `description` and `example` both of those operate in the same way `goasEnum` does.

Reply:
If I'm right about that being the convention, and if `description` and `example` are both introduced by `goas` and not used by anything else then yes, we should consider bringing them into the `goas` tag as well yeah, but doing so doesn't have to be in scope of this PR, we can circle back to that later.